core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java

```diff
@@ -140,7 +144,7 @@ public StackV4Response convert(StackDtoDelegate stack) {
         } catch (CloudbreakImageNotFoundException exc) {
             LOGGER.debug(exc.getMessage());
         }
-
+        response.setResources(null != stack.getResources() ? convertResources(stack.getResources()) : null);
```

Comment:
adding all resource to every stackresponse looks overkill, especially since everyone is using StackV4Response, maybe you should consider to create API to get resources only for a stack

Reply:
created a separate API

Reply:
done